src/couch/src/couch_debug.erl

```diff
@@ -75,29 +79,7 @@
 -spec help() -> [function_name()].
 
 help() ->
-    [
-        busy,
-        opened_files,
-        opened_files_by_regexp,
-        opened_files_contains,
-        process_name,
-        get_pid,
-        link_tree,
-        mapfold,
-        map,
-        fold,
-        linked_processes_info,
-        print_linked_processes,
-        memory_info,
-        print_table,
-        print_report,
-        print_report_with_info_width,
-        restart,
-        restart_busy,
-        dead_nodes,
-        ping_nodes,
-        node_events
-    ].
+    lists:uniq([F || {F, _A} <- couch_debug:module_info(exports)]) -- [help, module_info].
```

Comment:
Not every exported function has help. The idea of help function is to only return functions which can be called as `help(FunName)`. This is intended for higher level tools like admin GUI (not yet exists). The left panel would display list of topics (result of `help()`. call). By selecting one of the items the `help(SelectedItem)` is executed to display the help for that function. This change wold break this idea.